Samples/Samples/Samples/Views/EffectsLabelPage.xaml.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace FormsCommunityToolkit.Samples.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
Do we put the XAML compilation at the library level?

Reply:
Oh for F S.
Indeed we do, but thanks to the add new template in VS2017 it's being added each time on file level and I keep forgetting to delete it...
Good catch

Reply:
Did a new code push, that fixes this!